skopt/optimizer/base.py

```diff
@@ -222,9 +227,14 @@ def base_minimize(func, dimensions, base_estimator,
         if len(x0) != len(y0):
             raise ValueError("`x0` and `y0` should have the same length")
 
-        if not all(map(np.isscalar, y0)):
-            raise ValueError(
-                "`y0` elements should be scalars")
+        if acq_func not in ["EIps", "Pips"]:
+            if not all(map(np.isscalar, y0)):
+                raise ValueError(
+                    "`y0` elements should be scalars")
+        else:
+            if np.ndim(y0) != 2 and np.shape(y0)[1] != 2:
+                raise ValueError(
+                    "`y0` elements should be a tuple of 2 values.")
```

Comment:
Nitpick: one might get confused by this message by assuming that `y0` as such should be a tuple of 2 values. Maybe use something like "Every element in y0 should be a tuple or list containing 2 scalar values"